app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -31,40 +31,46 @@ import org.mozilla.fenix.theme.FirefoxTheme
 import org.mozilla.fenix.theme.Theme
 
 /**
- * Primary button.
+ * Base component for buttons.
  *
  * @param text The button text to be displayed.
- * @param icon Optional button icon to be displayed.
+ * @param text [Color] to apply to the button text.
+ * @param backgroundColor The background [Color] of the button.
+ * @param icon Optional [Painter] used to display a [Icon] before the button text.
+ * @param iconTint Tint [Color] to be applied to the icon.
  * @param onClick Invoked when the user clicks on the button.
  */
 @Composable
-fun PrimaryButton(
+private fun Button(
     text: String,
+    textColor: Color,
+    backgroundColor: Color,
     icon: Painter? = null,
+    iconTint: Color,
     onClick: () -> Unit,
 ) {
-    Button(
+    androidx.compose.material.Button(
         onClick = onClick,
         modifier = Modifier
             .clip(RoundedCornerShape(size = 4.dp))
```

Comment:
I'm seeing `RoundedCornerShape` in several places now. Long term, we may want to have a top-level Shapes Object which provides a one-stop-shop for things like this.

Reply:
Agreed, we should have something that feeds into `FirefoxTheme` similar to `MaterialTheme.shapes`. It's not totally clear what they all should be right now or what they should be named, but maybe as the component system is developed it will become clearer.

Reply:
It turns out we don't need to clip because of the Button defaults to `RoundedCornerShape(size = 4.dp)` https://cs.android.com/androidx/platform/tools/dokka-devsite-plugin/+/master:testData/compose/source/androidx/compose/material/Button.kt;l=99?q=Button.